includes/reader-revenue/woocommerce/class-woocommerce-cover-fees.php

```diff
@@ -132,25 +133,44 @@ public static function add_input_to_stripe_gateway_description( $desc ) {
 					value="true"
 					style="margin-right: 10px;"
 					onchange="newspackHandleCoverFees(this)"
+					<?php if ( get_option( 'newspack_donations_allow_covering_fees_default', false ) ) : ?>
+						checked
+					<?php endif; ?>
 				>
 				<label for=<?php echo esc_attr( self::CUSTOM_FIELD_NAME ); ?> style="display:inline;">
-					<b><?php echo esc_html( __( 'Cover transaction fees?', 'newspack-plugin' ) ); ?></b><br/>
 					<?php
+					$custom_message = get_option( 'newspack_donations_allow_covering_fees_label', '' );
+					if ( ! empty( $custom_message ) ) {
+						echo esc_html( $custom_message );
+					} else {
 						printf(
-							// Translators: %s is the transaction fee, as percentage with static portion (e.g. 2% + $0.3), %s is the site title.
-							esc_html__( 'Cover Stripe’s %1$s transaction fee, so that %2$s receives 100%% of your payment.', 'newspack-plugin' ),
-							self::get_fee_human_readable_value(), // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
-							esc_html( get_option( 'blogname' ) )
+							// Translators: %s is the possessive form of the site name.
+							esc_html__(
+								'I’d like to cover the %1$s transaction fee to ensure my full donation goes towards %2$s mission.',
+								'newspack-plugin' 
+							),
+							esc_html( self::get_fee_display_value() ),
+							esc_html( self::get_possessive( get_option( 'blogname' ) ) )
```

Comment:
Non-blocking: we're hardcoding English grammar here. We should change this eventually.

Reply:
Yes, I wasn't sure the best way to handle this. Wouldn't the placement of this as a placeholder in the `esc_html__` function make it possible for sites in other languages to use their own solution?

Reply:
It's added as an interpolated variable to the string, so it would output "Newspack's" regardless of the localized string.

Reply:
How should we solve this? I can't think of a solution to get around the silly English grammar rule.

Reply:
[This person](https://mattermost.com/blog/avoiding-common-internationalization-mistakes/) says `’s` is still correct with names ended in s, "according to other popular grammar guidelines". Looks weird to me, though.

It's not the prettiest, but we could make the transformation conditioned to `get_locale()`

Reply:
It is technically correct in English either way, but since it's related to the site's brand name I can see it being an issue if we allow `'s` with a site name ending in S. Let's leave it for now since the message itself is customizable.